<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="swWidgets">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pgLogin">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>143</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>300</height>
               </size>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="pixmap">
               <pixmap resource="resources.qrc">:/res/qtify.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="leftMargin">
               <number>6</number>
              </property>
              <item>
               <widget class="QLineEdit" name="leUsername"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lePassword">
                <property name="inputMethodHints">
                 <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText</set>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbRemember">
                <property name="text">
                 <string>Remember</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbLogin">
                <property name="text">
                 <string>Log in</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lbLoginError">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>134</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pgInside">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="PadNavigator" name="navigator"/>
            </item>
            <item>
             <widget class="QLineEdit" name="leSearch"/>
            </item>
            <item>
             <widget class="QPushButton" name="pbSearch">
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbSearch">
              <item>
               <property name="text">
                <string>Song</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Artist</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Album</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pgPlaylist">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="twPlaylist">
          <column>
           <property name="text">
            <string>#</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Track</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Artist</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Album</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Duration</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Popularity</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pgSearchSong">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="twSearchSong">
          <column>
           <property name="text">
            <string>#</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Track</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Artist</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Album</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Duration</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Popularity</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wPlayercontrols" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
       <item>
        <widget class="QPushButton" name="pbPrevious">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/res/prev.png</normaloff>:/res/prev.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbPlaypause">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/res/play.png</normaloff>:/res/play.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbNext">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/res/next.png</normaloff>:/res/next.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hsTimeline">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbShuffle">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/res/shuffleoff.png</normaloff>:/res/shuffleoff.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbBack">
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PadNavigator</class>
   <extends>QGraphicsView</extends>
   <header>padnavigator.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
